### PR TITLE
added who and what to readme

### PR DESCRIPTION
**Pull request by ghost**
Tried to make us look cooler

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
 # egg.spaces
 
-jvmaynard
-nimejia
-Cynabel
-sydward
+###Featured Developers :D
```
Comment (Hypaethral) [inline README.md]: Love this formatting, and good thinking added the links to github profiles

File: README.md [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+[nimejia](https://github.com/nimejia)
+[Cynabel](https://github.com/cynabel)
+[sydward](https://github.com/radiohYbrid)
+
+###Squadron Leaders <3
+
+[Hypaethral](https://github.com/Hypaethral)
+[dnaf](https://github.com/dnaf)
+
+###Who the heck are we?
```
Comment (Hypaethral) [inline README.md:15]: I like these descriptions, so wholesome!  I'm wondering if the "who we are" stuff should be part of the organization description, and we keep the readme specific to the project

**Review by Hypaethral — COMMENTED** [review]
Oh jeez asking for a review summary.. 

I like the additional formatting for featured developers!
I don't like the organization-level description in a specific project!

Good stuff, don't lose track of it :heart:

**Review by k80w — APPROVED** [review]
It's p lit

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md:15]: agreed. I'll take it out 😄

**Pull request closed without merging**